src/helpers/hotkeysManager.js

```diff
@@ -65,7 +64,15 @@ WebViewer(...)
     }
 
     // https://github.com/jaywcjlove/hotkeys#defining-shortcuts
-    hotkeys(key, handler);
+    const { keyup = () => {}, keydown = handler } = handler;
```

Comment:
Can we NOOP this function please. Besides that everything looks good to me. @ZhijieZhang this ones on you now!

Reply:
yeah sure. on it.